_docs/tasks/traffic-management-v1alpha3/ingress.md

```diff
@@ -47,15 +48,19 @@ This task describes how to configure Istio to expose a service outside of the se
 
 * Generate a certificate and key that will be used to demonstrate a TLS-secured gateway
 
-   A private key and certificate can be created for testing using [OpenSSL](https://www.openssl.org/).
+  A private key and certificate can be created for testing using [OpenSSL](https://www.openssl.org/).
 
-   ```bash
-   openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls.key -out /tmp/tls.crt -subj "/CN=foo.bar.com"
-   ```
+  ```bash
+  openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /tmp/tls.key -out /tmp/tls.crt -subj "/CN=foo.bar.com"
+  ```
 
 ## Configuring ingress using an Istio Gateway resource (recommended)
 
-> This is still a work in progress and is not yet functional.
+---
+
+> This is still a work in progress and is **not available** in the most recent Istio release (0.7.1). The current way of enabling inbound cluster traffic via Kubernetes Ingress resource is outlined [below](#configuring-ingress-using-a-kubernetes-ingress-resource).
```

Comment:
If you push this doc change in, it will only show up as part of the 0.8 docs. So is this warning still relevant in that context? 0.8 should be published next week.

We could apply this warning to the release-0.7 branch and push that out immediately to istio.io as well, if you'd like to submit a PR for that.

Reply:
Then I'm gonna close this and push to 0.7 instead — then, it's up to you guys, I just think that if 0.8 lags behind due to the fact you're dotting the *i*'s then this change would be a good patch in the meantime.

So sorry I haven't noticed the drastic spacing change!